### PR TITLE
Login page replace fetch 

### PR DESCRIPTION
**Pull request by patilnandini1811**

### CODE REVIEW

File: src/pages/LoginPage/LoginPage.jsx [hunk 1]
```diff
@@ -1,79 +1,71 @@
...
+        const user = mockUsers.find(user => user.username === username && user.password === password);
 
-	return (
-		<div>
-			<h1>Login Page</h1>
+        if (user) {
+            loginUser({ user: user.username }); 
+            setError("");
+					resetForm();
+					console.log(username);
```
Comment (KateN87) [inline src/pages/LoginPage/LoginPage.jsx]: Please remove this console.log

**Review by KateN87 — CHANGES_REQUESTED** [review]
Looks good, but please remove console.log

File: src/pages/LoginPage/LoginPage.jsx (hunk 1 above)
Comment (patilnandini1811) [reply src/pages/LoginPage/LoginPage.jsx]: Done!

**Review by KateN87 — APPROVED** [review]
looks good

**Pull request merged**